internal/commands/function/run_inputs.go

```diff
@@ -14,13 +14,14 @@ import (
 
 const (
 	flagFunctionName      = "name"
-	flagFunctionNameUsage = "specify the name of the function to run"
+	flagFunctionNameUsage = "Specify the name of the function to run"
 
-	flagFunctionArgs      = "args"
-	flagFunctionArgsUsage = "specify the args to pass to your function"
+	flagFunctionArgs = "args"
+	// TODO(REALMC-XXXX): Add docs link to the end of the usage string here: [Learn more: http://docs.link]
```

Comment:
⚠️  need to reach out to docs to get a JIRA to link to

Reply:
https://docs.mongodb.com/realm/functions/call-a-function/#call-from-realm-cli